droidlet/lowlevel/hello_robot/hello_robot_mover.py

```diff
@@ -182,28 +233,27 @@ def move_absolute(self, xyt_positions, blocking=True):
             xyt_positions = [xyt_positions]
         for xyt in xyt_positions:
             logging.info("Move absolute in canonical coordinates {}".format(xyt))
-            global_coords = base_canonical_coords_to_pyrobot_coords(xyt)
             self.is_moving.wait()
-            self.is_moving = self.bot.go_to_absolute(global_coords)
+            self.is_moving = self.bot.go_to_absolute(xyt)
             if blocking:
                 self.is_moving.wait()
         return "finished"
 
     def get_base_pos_in_canonical_coords(self):
         """get the current robot position in the canonical coordinate system
        
-        the standard coordinate systems:
-          Camera looks at (0, 0, 1),
-          its right direction is (1, 0, 0) and
-          its up-direction is (0, 1, 0)
+        the standard coordinate systems:                           
+        from the origin, at yaw=0, front is (x, y, z) = (1, 0, 0),
+        its right direction is (x,y,z) = (0, -1, 0) 
+        its up direction is (x,y,z) = (0, 1, 0) 
 
-         return:
-         (x, z, yaw) of the robot base in standard coordinates
+        return:
+        (x, z, yaw) of the robot base in standard coordinates
         """
         future = safe_call(self.bot.get_base_state)
         x_global, y_global, yaw = future.value
-        x_standard = -y_global
-        z_standard = x_global
+        x_standard = x_global
```

Comment:
So do we have a different standard coordinate system for hello? Why is that necessary?